README.md

```diff
@@ -154,7 +155,7 @@ existing users won't be broken.
 
 ### Kubernetes tags
 
-As of April 2018, client-go is still a mirror of
+client-go is still a mirror of
```

Comment:
I don't see us moving out of staging anytime soon. :)

Instead of updating the date here every time, we can remove this note if ever do so.

Reply:
This repository ....